Neos.Eel/Classes/Utility.php

```diff
@@ -54,8 +65,8 @@ public static function getDefaultContextVariables(array $configuration)
      */
     public static function evaluateEelExpression($expression, EelEvaluatorInterface $eelEvaluator, array $contextVariables, array $defaultContextConfiguration = [])
     {
-        $matches = null;
-        if (!preg_match(Package::EelExpressionRecognizer, $expression, $matches)) {
+        $eelExpression = self::parseEelExpression($expression);
+        if ($eelExpression === null) {
```

Comment:
@bwaidelich what if we just move the `throw` here into the `parseEelExpression` method? Then we get rid of the nullable return type.

Reply:
Yes, that's what I suggested. But we'd need a new exception (e.g. `NoEelExpressionException`) to differentiate it from exceptions that occur during evaluation of the Eel expression

Reply:
I wouldn't want to change the exception thrown here, since this is more a refactoring PR that simply extracts an internal functionality into a named method. But yes to new exception class for a broader rewrite of this Utility into a real Service.